cs_upload.py

```diff
@@ -209,6 +209,8 @@ def upload_to_s3(self, form_data: str, trace_file: str) -> None:
         for (key, value) in fields.items():
             query += ["--form", f"{key}={value}"]
         query += [
+            "--form",
+            "Content-Type=application/gzip",
```

Comment:
Couldn't this be an issue for uncompressed files?

For reference, here's what we done in cs-api: https://github.com/cryptosense/api-client/blob/54262ef9f04921a370374a913636175b7646f33d/cs_api_core/cs_api_core.ml#L138=.

Reply:
Fair point. It looks like the solution adopted in cs-api is to leave the `Content-Type` value empty (and not to provide a content type for the actual uploaded file in the `--form file=@foo.cst.gz;type=application/gzip` part). I think I'd prefer to provide a content type but have it depend on the filename extension: `.cst.gz` → `application/gzip` and `.cst` → `application/json`. What do you think?

Reply:
OK, how's this version?